docs/en/3_exist_data_new_model.md

```diff
@@ -41,6 +41,12 @@ mmdetection
 
 ```
 
+Or you can set your dataset root through
+```bash
```

Comment:
Need an empty line before and after the code block, according to the markdown style.

Reply:
> Need an empty line before and after the code block, according to the markdown style.

Thanks for your correction, I'll take care of it next time.
Thanks MMLAB, it's my first time to participate in open source project, and also my first PR.
Thanks @hhaAndroid @ZwwWayne  again, I'll review the mistake I took and be careful next time.
Thanks :)